### PR TITLE
enable debug logging for vrouter logs

### PR DESCRIPTION
**Pull request by anshprat**

### CODE REVIEW

File: manifests/vrouter.pp [hunk 1]
```diff
@@ -69,6 +72,7 @@
   $vgw_dest_net               = '0.0.0.0/0',
   $vgw_vrf                    = 'default-domain:services:public:public',
   $lbaas                      = true,
+  $debug_logging              = 1,
```
Comment (hkumarmk) [inline manifests/vrouter.pp]: Should this be true/false? Also this should be false by default.

Comment (hkumarmk) [inline manifests/vrouter.pp]: is $debug better variable for this?

**Comment by bodepd** [comment]
needs to be rebased, I've already verified it via a integration test and it's ready to be merged

**Comment by anshprat** [comment]
rebase done

**Comment by anshprat** [comment]
https://github.com/JioCloud/puppet-rjil/pull/627

File: manifests/vrouter.pp [hunk 2]
```diff
@@ -253,12 +268,22 @@
       require => Package['contrail-vrouter-agent'],
       notify  => Service['contrail-vrouter-agent'],
     }
+##
+# upstream conf file comment says only 0 (disable) or 1 (enable) for debug
+# instead of true/false. Note sure if true/false work properly/are supported
+##
+    if ($debug) {
```
Comment (hkumarmk) [inline manifests/vrouter.pp]: I just see there is one function available to convert boolean to number, so this is not required, instead you may just add for contrail_vrouter_config

'DEFAULT/debug': value => bool2num($debug)

**Comment by hkumarmk** [comment]
also I would just clean the history and would just make it one commit - for such a small change, it doesnt make sense to have 5 commits.

**Pull request merged**